test/e2e/service.go

```diff
@@ -275,9 +275,6 @@ var _ = Describe("Services", func() {
 			},
 		}
 
-		By("cleaning up previous service " + serviceName + " from namespace " + ns)
```

Comment:
Are we going to start leaking services target pools/forwarding rules? I don't see a defer statement to clean this up and we won't automatically clear it out when deleting the cluster.